integrationtest/neo/client/request/internal/DoublePut.d

```diff
@@ -150,69 +150,69 @@ public struct DoublePut
             start_request_on_new_conn();
 
         // Send to the selected node.
-        use_node(node,
```

Comment:
I don't think it's worth fixing closure allocations in tests.

Reply:
We discussed it - I can't distinguish between tests and non tests, this is the price of making the check non-ignorable. It is somewhat annoying but not that much.